core/graphelier-service/api/hndlrs/obhandler.go

```diff
@@ -0,0 +1,44 @@
+package hndlrs
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"graphelier/core/graphelier-service/models"
+
+	"github.com/gorilla/mux"
+)
+
+// JSONOrderbook : Sends an orderbook as json
+func JSONOrderbook(env *Env, w http.ResponseWriter, r *http.Request) error {
+	params := mux.Vars(r)
+	w.WriteHeader(http.StatusOK)
```

Comment:
Might be a bit premature setting the header to OK without completing the request

Reply:
good point